src/wkwebview/mod.rs

```diff
@@ -79,6 +81,30 @@ const NS_JSON_WRITING_FRAGMENTS_ALLOWED: u64 = 4;
 static COUNTER: Counter = Counter::new();
 static WEBVIEW_IDS: Lazy<Mutex<HashSet<u32>>> = Lazy::new(Default::default);
 
+#[derive(Debug)]
+pub struct PrintMargin {
+  pub top: f32,
+  pub right: f32,
+  pub bottom: f32,
+  pub left: f32,
+}
+
+impl Default for PrintMargin {
+  fn default() -> Self {
+    PrintMargin {
+      top: 0.0,
+      right: 0.0,
+      bottom: 0.0,
+      left: 0.0,
+    }
+  }
+}
```

Comment:
just derive, also derive Copy and Clone
```suggestion
#[derive(Debug, Default, Copy, Clone)]
pub struct PrintMargin {
  pub top: f32,
  pub right: f32,
  pub bottom: f32,
  pub left: f32,
}
```

Reply:
Changed on ca1ac37fc3ef8d4c4b31fe89dc0c22c56caed710